metadata.rb

```diff
@@ -8,3 +8,8 @@
 source_url 'https://github.com/chef-cookbooks/chef-ingredient'
 issues_url 'https://github.com/chef-cookbooks/chef-ingredient/issues'
 chef_version '>= 12.5' if respond_to?(:chef_version)
+
+supports 'debian'
+supports 'rhel'
+supports 'suse'
+supports 'windows'
```

Comment:
Are these _platforms_ or _platform families_? If it's the former we need to add:

* `centos`
* `freebsd`
* `mac_os_x`
* `solaris2`
* `ubuntu`

If it's the latter then we need to figure out how to represent these platforms as platform families. This cookbook essentially should support everything Chef supports.